UT4MasterServer/Services/AccountService.cs

```diff
@@ -25,18 +25,16 @@ public async Task CreateAccountAsync(string username, string email, string passw
 		Regex regex = new Regex(@"^([\w\.\-]+)@([\w\-]+)((\.(\w){2,3})+)$");
 		Match match = regex.Match(email);
 
-		if (!accountCollection.Find(email).Equals(true)||!match.Success)
+		if (!accountCollection.Find(email).Equals(true) || !match.Success)
 		{
 			await CreateAccountAsync(username, email, password);
 		}
```

Comment:
I fail to understand your logic here. why are you calling recursively the same method?
`Find` doesn't work like this. you can't just pass in whatever you want and expect it to find something.
You can see example usage of querying collections in other methods like `GetAccountAsync`.

Add a separate method for getting account by email `GetAccountByEmailAsync` and use that method to validate email in `AccountController.RegisterAccount`.